src/khoj/database/adapters/__init__.py

```diff
@@ -874,6 +883,7 @@ async def acreate_conversation_session(
     def create_conversation_session(
         user: KhojUser, client_application: ClientApplication = None, agent_slug: str = None, title: str = None
     ):
+        check_valid_user(user)
```

Comment:
We should probably make sure these errors are handled gracefully by relevant clients as well. E.g., in APIs, return an error with detail about what went wrong?

Reply:
Maybe we can do that as a follow-up? Sounds like a lot of work to track all the API endpoints these DB adapter functions are being called and put those calls into a try/catch block

For now clients will automatically see the API call fail with 500 internal server error. Server logs will show the issue. Which is a good enough start for now IMO?